doc/sphinx_pages/rest_sources/tutorials/concepts/misc/versatile_classification.tut

```diff
@@ -1,190 +0,0 @@
-
```

Comment:
Is this removed because it is not needed/wanted any more, or becaue it needs an update?

Reply:
i removed the versatile_* tutorials because they would need an update and a "look in how many different APIs you can classify" tutorial is maybe not very helpful. The idea in parts of the rework of the last months  was to make the APIs so consistent that you only need the distinction "do i train a neural network or can i use a premade trainer?" which made this obsolet